pkgs/applications/misc/zathura/default.nix

```diff
@@ -1,5 +1,5 @@
 { config, pkgs
-, useMupdf ? config.zathura.useMupdf or true
+, useMupdf ? config.zathura.useMupdf or (!pkgs.stdenv.isDarwin)
```

Comment:
Maybe make a comment that mupdf plugin is broken on darwin because it can't load a symbol at runtime (I guess from openjpeg?).

Reply:
Yeah might just be a bad link. Not required for merging, but if you could look at the output of:

```
$ otool -L /nix/store/27fc303khwdkqkbb3skr7dsrkdmlsld6-zathura-pdf-mupdf-0.3.4/lib/zathura/libpdf-mupdf.dylib
```

that would be helpful

Reply:
```
/nix/store/bbyhvy2yp7blcaai0cfkkgr59m6647lx-zathura-pdf-mupdf-0.3.4/lib/zathura/libpdf-mupdf.dylib:
        /nix/store/7c3nyrkwsfa2025kcf8v47x5hb0iykqi-girara-0.3.2/lib/libgirara-gtk3.3.dylib (compatibility version 3.0.0, current version 3.0.0)
        /nix/store/fb33y6z6ck2j2jmn69i9z7sbd7bx4iqr-cairo-1.16.0/lib/libcairo.2.dylib (compatibility version 11603.0.0, current version 11603.0.0)
        /nix/store/dj9ljyzlh6yzjpvg7hdm49ym8m7s0b9b-glib-2.60.1/lib/libglib-2.0.0.dylib (compatibility version 6001.0.0, current version 6001.1.0)
        /nix/store/g3c2jpj31bya2x3b8q50f5ix0lgalhgb-Libsystem-osx-10.12.6/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1226.10.1)
```